specs/protocol/configurability.md

```diff
@@ -30,47 +31,70 @@ There are four categories of OP Stack configuration options:
-| [Start block](https://github.com/ethereum-optimism/optimism/blob/c927ed9e8af501fd330349607a2b09a876a9a1fb/packages/contracts-bedrock/src/L1/SystemConfig.sol#L184)                           | Block at which the system config was initialized the first time.                                                             | [L1 Proxy Admin](#admin-roles)                      |
-| Superchain target                     | Choice of cross-L2 configuration. May be omitted in isolated OP Stack deployments. Includes SuperchainConfig and ProtocolVersions contract addresses. | Static |
+| Config Property                       | Description                                                                                                                  | Administrator                       | Standard Config Requirement | Notes |
+|---------------------------------------|------------------------------------------------------------------------------------------------------------------------------|-------------------------------------|-------------------------------------|-------------------------------------|
+| [Batch Inbox address](https://github.com/ethereum-optimism/optimism/blob/c927ed9e8af501fd330349607a2b09a876a9a1fb/packages/contracts-bedrock/src/L1/SystemConfig.sol#L176)                   | L1 address where calldata/blobs are posted (see [Batcher Transaction](../glossary.md#batcher-transaction)).                  | Static | Current convention is `0xff000...000{chainId}`. This doesn't work when `chainId` is `uint256`. | Follow convention if possible. A future hardfork plans to migrate Batch Inbox Addresses to better support `uint256` sized chain IDs. |
+| [Batcher Hash](./system_config.md#batcherhash-bytes32) | A versioned hash of the current authorized batcher sender(s). | [System Config Owner](#admin-roles) | `bytes32(uint256(uint160(batchSubmitterAddress)))` | [Batch Submitter](../protocol/batcher.md) address padded with zeros to fit 32 bytes. |
+| [Chain ID](https://github.com/ethereum-optimism/superchain-registry/blob/main/superchain/configs/chainids.json)                              | Unique ID of Chain used for TX signature validation.                                                                         | Static | Foundation-approved, non-colliding value [^chain-id]. | Foundation will ensure chains are responsible with their chain IDs until there's a governance process in place. |
+| [Challenge Period](../protocol/withdrawals.md#withdrawal-flow)                      | Length of time for which an output root can be removed, and for which it is not considered finalized.                        | [L1 Proxy Admin](#admin-roles)                      | 7 days | High security. Excessively safe upper bound that leaves enough time to consider social layer solutions to a hack if necessary. Allows enough time for other network participants to challenge the integrity of the corresponding output root. |
```

Comment:
> to a hack

This is not accurate, L1 will not bail out hacks. L1 will bail out censorship of fault proof game

Reply:
ah, I was reading this post: https://kelvinfichter.com/pages/thoughts/challenge-periods/ which says that this was the reason for the 7 days challenge period (among other things).